proto/v3/resolve.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+package datacommons.v3;
+
+option go_package = "github.com/datacommonsorg/mixer/internal/proto/v3";
+
+import "entity.proto";
+
+// Three modes are supported:
+//
+// 1. ID to ID:
+//   nodes = ["wikidataId/CA", "wikidataId/MA"]
+//   property = "<-wikidataId->nutsCode"
+//
+// 2. Coordinate (latitude#longitude) to DCID:
+//   nodes = ["34.56#-23.12", "-22.11#44.55"] 
+//   property = "<-geoCoordinate->dcid"
+//
+// 3. Description (name) to DCID:
+//   nodes = ["Mountain View", "New York"]
+//   property = "<-description->dcid" or "<-description{typeOf:City}->dcid"
+message ResolveRequest{
+    repeated string nodes = 1;
+    string property = 2;
+}
+
+message ResolveResponse{
+    message Entity {
+        string node = 1;
+        repeated datacommons.EntityInfo candidates = 2;
```

Comment:
I like the consistency that `EntityInfo` brings. One concern though is that it will break code like [this one](https://github.com/datacommonsorg/import/blob/43dbfac4e0a59d3498e201c4574cff9fa2b97f05/simple/util/dc_client.py#L143) that looks for the current fields.

Any thoughts on how we can address this? One approach could be to deprecate the current fields / messages and add new ones that use `EntityInfo`. Or another approach?

Reply:
Good point - discussed offline and will use the v2 proto for now (and can maybe migrate later if needed)